api/model.go

```diff
@@ -937,9 +957,21 @@ func volumeSchema(volume *client.Schema) {
 	unmapMarkSnapChainRemoved := volume.ResourceFields["unmapMarkSnapChainRemoved"]
 	unmapMarkSnapChainRemoved.Required = true
 	unmapMarkSnapChainRemoved.Create = true
-	unmapMarkSnapChainRemoved.Default = false
+	unmapMarkSnapChainRemoved.Default = longhorn.UnmapMarkSnapChainRemovedIgnored
```

Comment:
I believe this should be `longhorn.UnmapMarkSnapChainRemovedIgnored` string value instead of bool value `false`